### DISCUSSION TITLE
PS5 Card Configuration

### DISCUSSION THREAD

**Discussion by DAMIGUI** [FunkeyFlo/ps5-mqtt#275] [Q&A]
Hello there,

Could you share with us the configuration of your ps5 card? 
You've done a great job. And I'd like to integrate it into my HA.

Regards,

**Comment by FunkeyFlo** [comment]
As mentioned in the FAQ; no. Also, I don't use that config anymore.

The easiest way to get something similar is to use [Matt8707's config](https://github.com/matt8707/hass-config) and create a media player entity using the entities exposed by the addon. Guides on how to do the latter are available on the discord (link on main page).
